base/strings/annotated.jl

```diff
@@ -19,7 +19,7 @@ annotated with labeled values.
 ```
 
 The above diagram represents a `AnnotatedString` where three ranges have been
-annotated (labeled `A`, `B`, and `C`). Each annotation holds a label (`Symbol`)
+annotated (labelled `A`, `B`, and `C`). Each annotation holds a label (`Symbol`)
```

Comment:
These are not really typos but US vs UK way of writing.

Reply:
Ah, not this again... Thanks for letting me know, I'll just drop this commit.